{{ cookiecutter.repo_name }}/docs/user_guide/loading_environment_variables.md

```diff
@@ -1,8 +1,16 @@
 # Loading environment variables
 
-[We use `direnv` to load environment variables][direnv], as these are only loaded when
+[We use `python-dotenv` to load environment variables][python-dotenv], as these are only loaded when
 inside the project folder. This can prevent accidental conflicts with identically named
-variables.
+variables. Alternativley you can use [`direnv` to load environment variables][direnv] if
+you meet [certain conditions](#installing-direnv).
+
+## Using `python-dotenv`
```

Comment:
As above, we sometimes talk about os and sometimes python-dotenv

Reply:
So you need to use both to install. Here we have the heading python-dotenv to distinguish between the direnv method (the old method). As said earlier you use python-dotenv to load the .env script into python then we use os.getenv to select a specific value from that .env file.